Formula/c/croc.rb

```diff
@@ -24,7 +24,7 @@ def install
 
   test do
     # As of https://github.com/schollz/croc/pull/701 an alternate method is used to provide the secret code
-    ENV["CROC_SECRET"] = "homebrew-test" if OS.linux?
+    ENV["CROC_SECRET"] = "homebrew-test" if OS.linux? || OS.mac?
```

Comment:
```suggestion
    ENV["CROC_SECRET"] = "homebrew-test"
```

Reply:
that's actually better, since Homebrew is only for Linux and macOS. Thanks.